ros/src/actuation/vehicles/packages/as/nodes/ssc_interface/ssc_interface.cpp

```diff
@@ -202,13 +202,15 @@ void SSCInterface::publishCommand()
   unsigned char desired_mode = engage_ ? 1 : 0;
 
   // Speed for SSC speed_model
-  double desired_speed = vehicle_cmd_.ctrl_cmd.linear_velocity;
+  double desired_speed = vehicle_cmd_.twist_cmd.twist.linear.x;
 
   // Curvature for SSC steer_model
   double desired_steering_angle = !use_adaptive_gear_ratio_ ?
                                       vehicle_cmd_.ctrl_cmd.steering_angle :
                                       vehicle_cmd_.ctrl_cmd.steering_angle * ssc_gear_ratio_ / adaptive_gear_ratio_;
-  double desired_curvature = std::tan(desired_steering_angle) / wheel_base_;
+
+  double desired_curvature = vehicle_cmd_.twist_cmd.twist.angular.z / vehicle_cmd_.twist_cmd.twist.linear.x;
+  desired_curvature = desired_speed <= 0 ? 0 : desired_curvature;
```

Comment:
@cbradyas Good point about the 0 or negative speed. I had copied the original code from the old `pacmod_interface`:
https://github.com/autowarefoundation/autoware/pull/1945/files#diff-5d1010aa45e04c7cea3e2f0b75536a09L81

From what I've seen so far, autoware doesn't support going in reverse, although I could very well be wrong.
I think the idea for this line is to stop the steering wheel from trying to turn while the car is parked or motionless.

@snehagn are you aware of autoware being able to handle reverse? The PR you linked was closed and never merged.
Even if autoware could to reverse, the current ssc_interface only changes between `Gear::DRIVE` and `Gear::NONE`.